src/EditorFeatures/CSharpTest/AddUsing/AddUsingTests_Queries.cs

```diff
@@ -11,7 +11,7 @@ public partial class AddUsingTests
         [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsAddImport)]
         public async Task TestSimpleQuery()
         {
-            await TestInRegularAndScriptAsync(
+            await TestInRegular73AndScriptAsync(
```

Comment:
❔ Do we want _all_ of these tests to change? Or should we only change the ones that fail under C# 8.0?

Reply:
I'm preserving the existing behavior of the tests. There were too many failures to manually clean up.

Reply:
Looking at these, some of them makes sense that they're changing but a lot absolutely do not. We're going to have to look at these before this PR goes in at least to understand the root causes.

Reply:
Chatting with @agocke I see his intent was to baseline all the existing tests to their behavior. This isn't what we _want_ but is reasonable for him unblocking his own problems. I'm going to fork his PR to undo this rename to do the appropriate follow-up work.